libcontainer/configs/validate/validator.go

```diff
@@ -204,43 +222,18 @@ func (v *ConfigValidator) intelrdt(config *configs.Config) error {
 	return nil
 }
 
-func isSymbolicLink(path string) (bool, error) {
-	fi, err := os.Lstat(path)
-	if err != nil {
-		return false, err
-	}
+func isHostNetNS(path string) (bool, error) {
+	// assuming the current process is in initial netns
```

Comment:
s/initial/parent/

Reply:
I guess this depends on the definition of "host". I think of "host" as "the one that has initial namespaces".

Surely the current process is in the parent namespace, but is it in the host namespace? May be, may be not. What I meant by this is this function can give false positive (provided by definition of "host") in case runc is not in the host namespace.

Anyway, I think I was wrong, and in here by "host" we mean "current" or "parent to a container being created". In this case, it's better to just remove the comment to avoid confusion, or tell that here by "host" we mean "parent of a container being created".

Reply:
removed this comment, added another one elsewhere saying by "host" we mean "current" not "initial".